portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,49 +14,87 @@
 
 package com.google.sps.servlets;
 
-import com.google.sps.data.Comment;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import java.io.IOException;
+import com.google.sps.data.Comment;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
```

Comment:
General comments about the order of the imports (maybe a question for Kevin). Is there any style guide in Java to sort the imports alphabetically or other ways?

Reply:
Yes, imports should be sorted alphabetically.